packages/app/src/elements/item-view.ts

```diff
@@ -619,6 +642,44 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
+    private async handleDrop(event: DragEvent) {
+        event.preventDefault();
+
+        this._isDraggingFileToAttach = true;
```

Comment:
I considered adding a "uploading" view here, but:

1) We didn't have it for uploading via click/input.
2) It's barely visible since the dialog for the upload/file is on top.
3) There's already a notification when the upload finishes.